salt/metalk8s/addons/dex/deployed/clusterrolebinding.sls

```diff
@@ -0,0 +1,14 @@
+#!jinja | metalk8s_kubernetes kubeconfig=/etc/kubernetes/admin.conf&context=kubernetes-admin@kubernetes
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: dex-administrator
+subjects:
+- kind: User
+  name: "admin@metalk8s.invalid"
+  apiGroup: rbac.authorization.k8s.io
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
This file could have been added in another commit, or even another PR. But let's leave it, not critical.

Also, missing newline.